fzn-flatzingo.py

```diff
@@ -154,7 +154,7 @@ def main():
     if parsed.n is not None and optimization:
         raise Exception("Option -n is not allowed on optimization problems")
 
-    clingcon_command = ["clingcon", os.path.join(sys.path[0],"encoding.lp"), os.path.join(sys.path[0],"types.lp")]
+    clingcon_command = ["clingcon", os.path.join(sys.path[0],"encodings/encoding.lp"), os.path.join(sys.path[0],"encodings/types.lp")]
```

Comment:
You changed the command here but never used it in the readme.

Please take these comments as constructive. You did a lot in the readme and not all comments refer to your changes but to things that need to change in the future to make this a real/stable project.

Reply:
I didnt use it beacuse I never tried it. My guess was that this python file was to follow the standard of the compatitons. But please correct me if I am wrong and I add it to the readme as well. :)

Reply:
Yes, not only competition but also mini/flatzinc output format.

I also intend to create a minizinc solver file. This will ease a lot of things here, stay tuned...